apps/fhir/bluebutton/views/read.py

```diff
@@ -103,7 +112,7 @@ def read(request, resource_type, resource_id, *args, **kwargs):
     text_in = get_response_text(fhir_response=response)
     text_out = post_process_request(request, host_path, text_in, rewrite_url_list)
 
-    return JsonResponse(text_out)
+    return Response(text_out)
```

Comment:
This lets the response match the request, so if `application/json` is asked for that's what it returns, if `application/fhir+json` is asked for it returns that.

Reply:
Okay, sounds good. Do you want to update one of the tests to check that behavior (or at least one half). Just to make sure we're returning the response type. I feel like that's something that could break.